templates/download/alternative-downloads.html

```diff
@@ -6,137 +6,236 @@
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">
-          <a href="https://releases.ubuntu.com/{{ releases.previous_lts.full_version }}/">Ubuntu {{
-            releases.previous_lts.short_version }} LTS ({{ releases.previous_lts.name }})</a>
-        </li>
-        {% if releases.latest.past_eol_date != true and releases.latest.short_version < releases.lts.short_version %}
-          <li class="p-list__item is-ticked">
-          <a href="https://releases.ubuntu.com/{{ releases.latest.full_version }}/">Ubuntu {{
-            releases.latest.short_version }} ({{ releases.latest.name }})</a>
-          </li>
-          {% endif %}
-          <li class="p-list__item is-ticked">
-            <a href="https://releases.ubuntu.com/{{ releases.previous_previous_lts.full_version }}/">Ubuntu {{
-              releases.previous_previous_lts.short_version }} LTS ({{ releases.previous_previous_lts.name }})</a>
-          </li>
-      </ul>
-      <p><a href="https://releases.ubuntu.com/">Past releases</a></p>
+      <div class="p-section">
+        <div class="row">
+          <hr class="p-rule">
+          <div class="col-7">
+            <div class="p-section--shallow">
+              <h2 id="network-installer" class="question-heading" style="scroll-margin-top: 3.4rem;">Network installer</h2>
+            </div>
+            <p>The network installer lets you install Ubuntu over a network. It includes the minimal set of packages needed to start and the rest of the packages are downloaded over the network. Since only current packages are downloaded, there is no need to upgrade packages immediately after installation.</p>
+            <p>The network installer is ideal if you have a computer that cannot run the graphical installer. It is also useful if you want to install Ubuntu on a large number of computers at once.</p>
+            <hr class="p-rule is-muted">
+            <p><a href="https://discourse.ubuntu.com/t/netbooting-the-live-server-installer?_gl=1*1t2mr3n*_gcl_au*MTE4NTIyOTI0MS4xNzA3MTMxMDQx&_ga=2.204825875.2084151835.1707729318-1126754318.1683186906">Instructions for the 22.04 LTS Ubuntu Live installer&nbsp;&rsaquo;</a></p>
+            <hr class="p-rule is-muted">
+            <p><a href="https://discourse.ubuntu.com/t/netbooting-the-live-server-installer">Instructions for the 20.04 Ubuntu Live installer&nbsp;&rsaquo;</a></p>
+            <hr class="p-rule is-muted">
+            <p><a href="http://cdimage.ubuntu.com/netboot/18.04/">Download the network installer for 18.04 LTS&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
     </div>
   </div>
 </div>
 
-{% with first_item="_download_more_info_enterprise", second_item="_download_all_installation",
-third_item="_download_helping_hands" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+{% include "download/shared/_footer.html" %}
+
+<script type="text/javascript">
+  function setUpTogglableSideNav() {
```

Comment:
Why not put this in side-nav.js? There's a function in there that looks to be doing a very similar thing https://github.com/canonical/ubuntu.com/blob/95aa5d69d08dbbedecea0748d6944e76f9fff266/static/js/src/side-navigation.js#L90